libutils/buffer.c

```diff
@@ -32,7 +32,7 @@
 #endif
```

Comment:
@larsewi This commit doesn't have any user-facing changes, so there should not be a changelog entry.

https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#changelog